git-appraise/git-appraise.go

```diff
@@ -86,6 +78,19 @@ func main() {
 		fmt.Printf("%s must be run from within a git repo.\n", os.Args[0])
 		return
 	}
+	if len(os.Args) < 2 {
+		subcommand, ok := commands.CommandMap["list"]
+		if !ok {
+			fmt.Printf("Unable to list reviews")
+			return
+		}
+		subcommand.Run(repo, []string{})
+		return
+	}
+	if os.Args[1] == "help" {
+		help()
```

Comment:
I think `git appraise help` should still work even if the current directory is not inside of a git repository.

Perhaps make this something like:

    if len(os.Args) > 0 && os.Args[1] == "help" {

... and move this block to line 71